dnf-behave-tests/dnf/module/install-package-ursine.feature

```diff
@@ -32,7 +32,7 @@ Scenario: I can't reinstall installed non-modular content which is masked by act
     And I successfully execute dnf with args "module enable DnfCiModuleNoDefaults:stable"
    When I execute dnf with args "reinstall wget"
    Then the exit code is 1
-    And stderr contains "Packages for argument 'wget' installed and available, but in a different version => cannot reinstall"
+    And stderr contains "Installed packages for argument 'wget' are not available in repositories, available versions: wget-1.18.5-5.module.x86_64, cannot reinstall."
```

Comment:
I think it's a bit confusing that the message says that the packages are not available. It would be better if it said "are not available in repositories in the same version".

Reply:
That is better, updated, thanks!